### PR TITLE
Add skos definitions

### PR DESCRIPTION
**Pull request by rjyounes**
Added two additional skos definitions, for bdo:Gender and bdo:personGender.

### CODE REVIEW

**Comment by rjyounes** [comment]
@eroux Just a nudge on this one.

**Review by eroux — CHANGES_REQUESTED** [review]

**Comment by eroux** [comment]
oops, I commented a while ago but it seems I haven't clicked on the right button sorry!

File: core/bdo.ttl [hunk 1]
```diff
@@ -481,6 +481,7 @@ Often the Title Page Title for a publication of a reproduction that includes pre
 
 bdo:Gender
 	rdfs:subClassOf bdo:Type ;
+	skos:definition "A category into which humans are divided on the basis of their reproductive functions."@en ;
```
[the remark this answers is not in the document]
Comment (rjyounes) [reply core/bdo.ttl:484]: Is the difference between biological sex and gender identity ever relevant in this knowledge domain?

[the remark this answers is not in the document]
Comment (eroux) [reply core/bdo.ttl:484]: I can't think of any case where we would want to assert something terribly specific here... I think we would be safe with an inclusive definition

**Review by eroux — CHANGES_REQUESTED** [review]
Thanks! The skos definitions are good, can you put the changes to the scripts in a separate PR? (Also they look a bit off)

**Pull request closed without merging**